### DISCUSSION TITLE
Homepage path corrupted after change to Custom navigation

### DISCUSSION THREAD

**Discussion by mittkkoo** [requarks/wiki#7266] [Error / Bug Report] [answered]
I changed the navigation to Custom and suddenly the path of the homepage changed from `/home` to `/manage:system`. I wasn't able to repair this from the UI. The button to move the page was grayed out and also the edit mode suggested to create the page as if it was not existing at all. So I went to the PostgreSQL database and found the page by its id. It was still there and the content was all right. I changed the path in the database from  `manage:system` to `home`. The result is that the homepage is back, but 2 things are missing:
- The navigation on the homepage is empty and there is a spinning circle in the header bar.
- On other pages the navigation is all right, however the link to the homepage targets `manage:system`, which doesn't exist. The link on the logo is OK.

I'm have one locale, if relevant. The page history tracked an edit on the homepage done by the Admin at the time the bug occured, however didn't show the change and I also didn't find anything else in the database.

![image](https://github.com/user-attachments/assets/db579df8-a59c-4437-91a5-9cc52fad3912)

2.5.298
Docker Container (Linux)
Hostname: wiki
CPU: 2
RAM: 3.84 GB
/wiki
/wiki/config.yml
PostgreSQL 11.16 db
Node.js 16.19.1

**Comment by NGPixel** [comment, marked as answer]
Changing navigation modes cannot result in the homepage being renamed. There's simply no logic in the code to do that. You or someone else with editing rights have accidentally renamed the page.

You'll need to rebuild the page tree. Modifying the page path by itself is not sufficient. Make any edit to any page to trigger a page tree rebuild.

**Reply by mittkkoo** [reply]
Thanks!
I had to move some page for the tree to rebuild. Only editing the content didn't do it. After that everything is OK now.
However the bug remains. From the UI it is not possible to move the homepage to an unexisting path. If moved, at least the new path is working, I tried it. Something else must have triggered it.
